index.html

```diff
@@ -0,0 +1,97 @@
+        <h4>Here's a time line of Dr. Borlaug's life:</h4>
+        <ul class="years">
+          <li><strong>1914</strong> - Born in Cresco, Iowa</li>
+          <li>
+            <strong>1933</strong>- Leaves his family's farm to attend the
+            University of Minnesota, thanks to a Depression era program known as
+            the "National Youth Administration"
+          </li>
+          <li>
+            <strong>1935</strong>- Has to stop school and save up more money.
+            Works in the Civilian Conservation Corps, helping starving
+            Americans. "I saw how food changed them", he said. "All of this left
+            scars on me."
+          </li>
+          <li>
+            <strong>1937</strong>- Finishes university and takes a job in the US
+            Forestry Service
+          </li>
+          <li>
+            <strong>1938</strong>- Marries wife of 69 years Margret Gibson. Gets
+            laid off due to budget cuts. Inspired by Elvin Charles Stakman, he
+            returns to school study under Stakman, who teaches him about
+            breeding pest-resistent plants.
+          </li>
+          <li>
+            <strong>1941</strong> - Tries to enroll in the military after the
+            Pearl Harbor attack, but is rejected. Instead, the military asked
+            his lab to work on waterproof glue, DDT to control malaria,
+            disinfectants, and other applied science.
+          </li>
+          <li>
+            <strong>1942</strong>- Receives a Ph.D. in Genetics and Plant
+            Pathology
+          </li>
+          <li>
+            <strong>1944</strong>- Rejects a 100% salary increase from Dupont,
+            leaves behind his pregnant wife, and flies to Mexico to head a new
+            plant pathology program. Over the next 16 years, his team breeds
+            6,000 different strains of disease resistent wheat - including
+            different varieties for each major climate on Earth.
+          </li>
+          <li>
+            <strong>1945</strong> Discovers a way to grown wheat twice each
+            season, doubling wheat yields
+          </li>
+          <li>
+            <strong>1953</strong>- crosses a short, sturdy dwarf breed of wheat
+            with a high-yeidling American breed, creating a strain that responds
+            well to fertilizer. It goes on to provide 95% of Mexico's wheat.
+          </li>
+        </ul>
+      </div>
+
+      <h5>
+        If you have time, you should read more about this incredible human being
+        on his
+        <a id="tribute-link" href="https://en.wikipedia.org/wiki/Norman_Borlaug" target="_blank"
+          >Wikipedia entry.</a
+        >
+      </h5>
+
+      <p><em>"Borlaug's life and achievement are testimony to the far-reaching contribution that one man's towering intellect, persistence and scientific vision can make to human peace and progress."
+
+        -- Indian Prime Minister Manmohan Singh</em></p>
```

Comment:
Please group this part of your code. It should be easy for the reader of the code to understand what is this text for and easier for you to style it.